x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -231,6 +231,29 @@ event action.
 ["index-b*"],"privileges":["all"]}],"applications":[],"run_as":[]}]}}}
 ====
 
+[[event-update-apikey]]
+`update_apikey`::
```

Comment:
We might want change this to `change_apikey` since `change` seems to be the word of choice with the current naming convention for audit events. @albertzaharovits Your view is appreciated.

Reply:
I'm good with `change_apikey`. I went with `update_apikey` based on `update_user_profile_data` but no strong preference either way.

Reply:
I've updated to `change_apikey`, provisionally.

Reply:
The current convention for auditing security config changes, is that for `event.action`  of type `x_y`, the audited config content starts with `"x": { "y":" (where is `x` can be one of `put`, `delete`, `change`, `create` or `invalidate`). Accoding to that, `change_apikey` goes with `"change": {"api_key"`.

Reply:
Going with `change_apikey`. Thanks!